src/MongoDB.Driver/MongoClient.cs

```diff
@@ -67,6 +67,10 @@ public MongoClient()
 
         /// <summary>
         /// Initializes a new instance of the MongoClient class.
+        /// In .NET Standard, authenticating via SCRAM-SHA-256 may not work with non-ASCII passwords because SaslPrep is
+        /// not fully implemented due to the lack of a string normalization function in .NET Standard 1.6.
+        /// Normalizing the password into Unicode Normalization Form KC beforehand MAY help.
+        /// SCRAM-SHA-1 is the recommended alternative for now.
```

Comment:
I was expecting this warning somewhere more precise like when setting a password or select SCRAM-SHA-256 authentication.